### PR TITLE
chore: Add documentation for common POST */react errors

### PR DESCRIPTION
**Pull request by koalaty-code**
## Description

- Adds documentation for Reactor errors and common scenarios.

## Testing required outside of automated testing?

- [X] Not Applicable

### Screenshots (if appropriate):

- [X] Not Applicable

## Rollback / Rollforward Procedure

- [X] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: source/includes/api-reference/_atomic_banks.md [hunk 1]
```diff
@@ -461,7 +461,7 @@ Parameter | Required | Type | Default | Description
 ### Response
 
 Returns a [token](#token-object) with type of `bank:reaction` if the Atomic Bank was reacted. Returns [an error](#errors) if the Atomic Bank failed to react.
-
+Third-party errors will be translated to the common Basis Theory Error format. See [React Errors](#react-errors) for more details.
```
Comment (armsteadj1) [inline source/includes/api-reference/_atomic_banks.md:464]:
```suggestion
Errors generated from Reactors will be translated to the common Basis Theory Error format. See [React Errors](#react-errors) for more details.
```

File: source/includes/api-reference/_atomic_cards.md [hunk 2]
```diff
@@ -500,7 +500,7 @@ Parameter | Required | Type | Default | Description
 ### Response
 
 Returns a [token](#token-object) with type of `card:reaction` if the Atomic Card was reacted. Returns [an error](#errors) if the Atomic Card failed to react.
-
+Third-party errors will be translated to the common Basis Theory Error format. See [React Errors](#react-errors) for more details.
```
Comment (armsteadj1) [inline source/includes/api-reference/_atomic_cards.md:503]:
```suggestion
Errors generated from Reactors will be translated to the common Basis Theory Error format. See [React Errors](#react-errors) for more details.
```

File: source/includes/api-reference/_errors.md [hunk 3]
```diff
@@ -52,3 +55,21 @@ Error Code | Meaning
 `422` | Request does not satisfy requirements for processing
 `429` | Request has been [rate limited](#limits)
 `500` | Something went wrong on Basis Theory's side
+
+
+## React Errors
```
Comment (dhudec) [inline source/includes/api-reference/_errors.md:60]: Should we be calling these `Reactor Errors` or `React Errors`? I could see either way, but something about React Error sounds weird to me.

Comment (koalaty-code) [reply source/includes/api-reference/_errors.md:60]: I will rename and update the anchor to use Reactor Errors.

**Review by dhudec — APPROVED** [review]

**Review by brigonzalez — APPROVED** [review]

**Pull request merged**